pkg/cmd/server/server.go

```diff
@@ -307,18 +307,12 @@ func (s *server) run() error {
 		return err
 	}
 
-	if err := s.veleroResourcesExist(); err != nil {
-		return err
-	}
-
-	if err := s.validateBackupStorageLocations(); err != nil {
-		return err
-	}
-
-	if _, err := s.veleroClient.VeleroV1().BackupStorageLocations(s.namespace).Get(s.config.defaultBackupLocation, metav1.GetOptions{}); err != nil {
-		s.logger.WithError(errors.WithStack(err)).
-			Warnf("A backup storage location named %s has been specified for the server to use by default, but no corresponding backup storage location exists. Backups with a location not matching the default will need to explicitly specify an existing location", s.config.defaultBackupLocation)
-	}
+	// Anything that could be created after Velero has started will be
+	// queried as part of the liveness check
+	go func() {
+		s.logger.Info("Starting liveness server at port 2000")
```

Comment:
It would be nice to have the port number "2000" either as a constant or configurable via variable.